templates/cassiopeia/index.php

```diff
@@ -114,6 +118,8 @@
 	. ($task ? ' task-' . $task : ' no-task')
 	. ($itemid ? ' itemid-' . $itemid : '')
 	. ($pageclass ? ' ' . $pageclass : '')
+    . ($breakPointTemplate ? ' break-tmpl-' . $breakPointTemplate : '')
+    . ($breakPointGrid ? ' break-grid-' . $breakPointGrid : '')
```

Comment:
```suggestion
	. ($breakPointTemplate ? ' break-tmpl-' . $breakPointTemplate : '')
	. ($breakPointGrid ? ' break-grid-' . $breakPointGrid : '')
```
Please use tabs to indent PHP code.

Reply:
Fix in my next commit